docs/admin/admission-controllers.md

```diff
@@ -568,8 +568,7 @@ This admission controller will deny any pod that attempts to set certain escalat
 This admission controller implements automation for [serviceAccounts](/docs/user-guide/service-accounts).
 We strongly recommend using this admission controller if you intend to make use of Kubernetes `ServiceAccount` objects.
 
-### Storage Object in Use Protection (beta)
-{% assign for_k8s_version="v1.10" %}{% include feature-state-beta.md %}
+### Storage Object in Use Protection
```

Comment:
Do we want to mention that this is a feature formerly known as PVC Protection? Or that it's called PVC Protection in earlier versions of K8s?

Reply:
Firstly, IMHO, users of K8s 1.11 documentation are not interested in K8s history, I think they are only interested in K8s 1.11.
I believe that users interested in history, i.e. differences between particular K8s versions will read release notes of these K8s versions.
That's why I think we don't need to mention that `Storage Object in Use` feature was in K8s 1.9 implemented only partially and was called `PVC Protection`.

---

Lastly, @Bradamant3 your question:
> Same perhaps for the related tasks topic?

Makes me think of the following: `Storage Object In Use` feature currently protects PVCs and PVs. It may protect other Storage Objects in future. Is this clear from the documentation?